layouts/_default/baseof.html

```diff
@@ -73,7 +73,7 @@
       height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
     <!-- End Google Tag Manager -->
 
-    <nav>
+    <nav aria-label="{{ i18n "skip-links" }}">
```

Comment:
I'm not sure how the aria-label plays a role here. Is it supposed to be read by some screen readers? If yes, does it allow to skip the navigation? Using Voiceover on chrome, I couldn't get it to say "Skip navigation links" when going from element to element using VO+arrows.

Reply:
Oh, nevm, I see we have more than one nav element. And the label lets us know which nav this is